feature-configs/cn-ran-overlays/README.md

```diff
@@ -0,0 +1,22 @@
+# cn-ran-overlays
+
+This folder contains an example configurations for 5G DU site
+(CU - work in progress)
```

Comment:
I think er should state that a CU is the same profile with the exception that:
1.  Enable a stock kernel instead of an rt-kernel.
2.  Skip PTP
3.  Skip FEC
4.  Add NTP (this is a WIP).